docs-beta/content/getting_started/kube.md

```diff
@@ -0,0 +1,217 @@
+Because of this, the included manifests reference a StorageClass named fast. Manifests are provided to provide such a StorageClass on AWS / Azure / GCP using the respective cloud provider's premium disk class.
+If you do not already have a StorageClass named fast, create one using one of the provided manifests:
+#### AWS EBS (class io1)
+kubectl apply -f https://raw.githubusercontent.com/m3db/m3/master/kube/storage-fast-aws.yaml
+
+#### Azure premium LRS
+kubectl apply -f https://raw.githubusercontent.com/m3db/m3/master/kube/storage-fast-azure.yaml
+
+#### GCE Persistent SSD
+kubectl apply -f https://raw.githubusercontent.com/m3db/m3/master/kube/storage-fast-gcp.yaml
+
+If you wish to use your cloud provider's default remote disk, or another disk class entirely, you'll have to modify them manifests.
+If your Kubernetes cluster spans multiple availability zones, it's important to specify a Volume Binding Mode of WaitForFirstConsumer in your StorageClass to delay the binding of the PersistentVolume until the Pod is created.
+
+**Kernel Configuration**
+We provide a Kubernetes daemonset that can make setting host-level sysctls easier. Please see the kernel docs for more.
+Note that our default StatefulSet spec will give the M3DB container CAP_SYS_RESOURCE so it may raise its file limits. Uncomment the securityContext on the m3db container in the StatefulSet if running with a Pod Security Policy or similar enforcement mechanism that prevents adding capabilities to containers.
+Deploying
+Apply the following manifest to create your cluster:
+kubectl apply -f https://raw.githubusercontent.com/m3db/m3/master/kube/bundle.yaml
+
+Applying this bundle will create the following resources:
+An m3db Namespace for all M3DB-related resources.
+A 3-node etcd cluster in the form of a StatefulSet backed by persistent remote SSDs. This cluster stores the DB topology and other runtime configuration data.
+A 3-node M3DB cluster in the form of a StatefulSet.
+Headless services for the etcd and m3db StatefulSets to provide stable DNS hostnames per-pod.
+Wait until all created pods are listed as ready:
+$ kubectl -n m3db get po
+NAME         READY     STATUS    RESTARTS   AGE
+etcd-0       1/1       Running   0          22m
+etcd-1       1/1       Running   0          22m
+etcd-2       1/1       Running   0          22m
+m3dbnode-0   1/1       Running   0          22m
+m3dbnode-1   1/1       Running   0          22m
+m3dbnode-2   1/1       Running   0          22m
+
+You can now proceed to initialize a namespace and placement for the cluster the same as you would for our other how-to guides:
+#### Open a local connection to the coordinator service:
+$ kubectl -n m3db port-forward svc/m3coordinator 7201
+Forwarding from 127.0.0.1:7201 -> 7201
+Forwarding from [::1]:7201 -> 7201
+
+#### Create an initial cluster topology
+curl -sSf -X POST localhost:7201/api/v1/placement/init -d '{
+    "num_shards": 1024,
```

Comment:
I think the number of shards here should be 64?

Reply:
@xcode03 - thanks! We'll take a look.